meta-oe/recipes-devtools/pahole/pahole_1.22.bb

```diff
@@ -0,0 +1,24 @@
+SUMMARY = "Shows and manipulates data structure layout"
+
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://COPYING;md5=751419260aa954499f7abaabaa882bbe"
+
+DEPENDS = "elfutils zlib libbpf"
+
+SRCREV = "f02af2553ea58ae1186226af0d0ec835a248358f"
+SRC_URI = "git://git.kernel.org/pub/scm/devel/pahole/pahole.git \
+	   file://0001-CMakeList.txt-make-python-optional.patch"
+
+S = "${WORKDIR}/git"
+
+inherit cmake
+
+PACKAGECONFIG[python3] = ",,python3-core,python3-core"
+
+EXTRA_OECMAKE = "-D__LIB=lib -DCMAKE_BUILD_TYPE=Release -DLIBBPF_EMBEDDED=OFF"
+
+FILES_${PN} += "${datadir}"
+
+RDEPENDS_${PN} += "bash"
```

Comment:
is this must ? can be separate the bash scripts into a separate package ?

Reply:
I could put only the `pahole` binary in the pahole package, and all the other stuff in `pahole-extra`.
what do you think?

Reply:
yeah sounds good.